service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/prepopulator/question/ChildPlacementOrderPrePopulatorTest.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.fpl.service.orders.prepopulator.question;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.service.PlacementService;
+import uk.gov.hmcts.reform.fpl.utils.assertions.DynamicListAssert;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.Constants.TEST_CASE_ID;
+import static uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock.CHILD_PLACEMENT_APPLICATIONS;
+
+@ExtendWith(MockitoExtension.class)
+class ChildPlacementOrderPrePopulatorTest {
+
+    private final UUID mockedChildApplicationId = UUID.randomUUID();
+    private Element<String> mockedChildApplication = Element.<String>builder()
```

Comment:
you can use element(mockedChildApplicationId, "Mocked child application"); 

from ElementUtils